www/perl-lib/SMT/Rest/SCCAPIv4.pm

```diff
@@ -630,6 +630,10 @@ sub _calcMigrationTargets
         }
         push @result, \@productset;
     }
+
+    # Remove duplicate migration paths (bsc#1097824)
```

Comment:
just curious: why we mention bug in the code (we have it already in changelog) ?

Reply:
So that whoever descends into this abyss of insanity in the future has an easier time figuring out what's going on, especially since it's a different implementation in a different language. :grimacing: 
(The changelog is for the users, the comments in the code are for the developers)